src/EnergyPlus/StandardRatings.cc

```diff
@@ -519,14 +520,15 @@ namespace StandardRatings {
         //           551/591 conditons[C]
         // par(2)  = Evaporator outlet temperature [C]
         // par(3)  = Water specific heat [J/(kg*C)]
-        // par(4)  = Part load ratio
-        // par(5)  = Evaporator mass flow rate [kg/s]
+        // par(4)  = Load
```

Comment:
Pass the percentage load instead of the PLR, so the PLR can be calculated based on the CAP-f-T.